cliquet/views/errors.py

```diff
@@ -57,19 +57,28 @@ def page_not_found(request):
     redirect_enabled = request.registry.settings[config_key]
     trailing_slash_redirection_enabled = asbool(redirect_enabled)
```

Comment:
I have added that here: https://github.com/Kinto/kinto/pull/583
I realized that a lot are missing in cliquet docs too.